### PR TITLE
Emphasised need for memcached

### PR DESCRIPTION
**Pull request by Nolski**
This seemed pretty easy to miss but since the wording was there I think  it might be a good idea to emphasize this text.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ For security, you need to enter something into the `SECRET_KEY`.
 SECRET_KEY = 'somethingnotempty'
 ```
 
-By default, you need a `Memcache` server up and running. The connection
+*By default, you need a `Memcache` server up and running.* The connection
```
Comment (peterbe) [inline README.md]: Italic text is for emphasis on a specific word like "The cat was in the *house* not in the barn". 
Use bold for things that are supposed to stand out.

Comment (Nolski) [inline README.md]: For code review changes, would you rather I add an additional commit with the fix or amend my previous commit?

**Comment by Nolski** [comment]
updated to be bold.

File: README.md (hunk 1 above)
Comment (peterbe) [inline README.md]: Keep adding commits until you get an r+ comment. Then, when you have one of those, go ahead and rebase your PR down to 1 neat messaged commit and force push that into your branch and once the travis tests have tested one last time, I merge.

**Pull request merged**